docs/Course Wiki/PHYS Course Offerings/PHYS212.md

```diff
@@ -0,0 +1,59 @@
+# PHYS212 
+
+PHYS212(University Physics: Electricity and Magnetism) is a 4-credit hour course that is required for ECE majors as a part of the Foundational Mathematics and Science requirement. It is also required for most other engineering majors, including Computer Science majors. It is offered in the fall, spring, and summer semesters.
+
+## Content Covered
+
+- Coulomb’s Law, Electrostatics, Electric Field
+- Electric Flux, Gauss’s Law, Electric Potential
+- Capacitors and Capacitance
+- Current and DC Circuit Analysis, KVL, KCL, Ohm’s Law
+- RC Circuits
+- Magnetic Force and Field, Biot-Savart and Ampere’s Law
+- Magnetic Induction, Faraday’s Law, Motional EMF
+- Inductors and RL Circuits
+- LC and LRC Circuits, Harmonic Oscillation, Phasors
+- AC Circuits, Resonance, Power, Transformers
+- Maxwell’s Equations, Electromagnetic Waves
+- Polarization, Intensity, Geometric Optics
+
+PHYS212 is a solid foundational course in many concepts absolutely essential for any ECE major. Concepts such as DC circuit analysis, phasors, resonance and power, and basic electromagnetism provide an excellent conceptual base for ECE majors to understand topics in courses such as ECE 110, 210, 330, and more.
+
+## Prerequisites
+
+The official prerequisite for PHYS212 is PHYS211. 
+
+## When to Take It
+
+ECE majors typically take PHYS212 in their freshman year, most commonly in spring after doing PHYS211 in the fall. It is a course that is quite helpful for ECE 110 and 210 if the student understands the concepts properly. The student will then find that it provides a nice scientific basis and intuition for a lot of ideas in 110 and 210 that are merely presented axiomatically as equations to be used in circuit analysis. 
```

Comment:
Add a sentence about how it is an important prerequisite for ECE210, so it should not be put off.  Also, link the wiki pages for courses (even if they don't exist yet) the first time they are mentioned in a section, this goes for each section.